.github/ISSUE_TEMPLATE/02_feature_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Feature Request
+description: Enchancement to admiral functionality
+title: "Feature Request:  <Insert Issue Title Here>"
+labels: ["enhancement", "programming"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!  We love keeping admiral fresh!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: feature
+    attributes:
+      label: Feature Idea (Succint)?
+      description: Tell us your idea in as few words as possible
+      placeholder: Tell us what you see!
+      value: "`derive_var_dtm` should do x, y and z "
+    validations:
+      required: true
+  - type: textarea
+    id: output
+    attributes:
+      label: Relevant Output
```

Comment:
What about the input?

Just an example output may be hard to understand without the corresponding input.

Reply:
Yes good point!